functions/Test-DbaLastBackup.ps1

```diff
@@ -106,6 +106,9 @@ function Test-DbaLastBackup {
         This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
         Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
 
+    .PARAMETER MaxDop
```

Comment:
Can you reorder the parameter help to match that of the command? 🥺 

I think mostly we have the following at the end (in this order), then all the command-specific parameters go above it:
- InputObject
- WhatIf
- Confirm
- EnableException